test/integration/host_test.rb

```diff
@@ -413,15 +413,17 @@ def test_show_action_buttons
   end
 
   describe 'clone page' do
-    test 'shows no errors on lookup values' do
```

Comment:
Why remove this test?  Add a `skip` if it's known to be broken and file a bug?

The bug it's testing for is actually present now (since inline editing), but the idea it's testing is still relevant I think.